db/migrate/20230721125944_add_users_ref_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsersRefToComments < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :comments, :users, null: false, foreign_key: true
```

Comment:
- [ ] Kindly consider changing this migration because as in the EDR diagram, you should have `author_id` as foreign_key and not `user_id` in your Comments table. All you need to do is to have an `author_id` referencing User table instead of `user_id`.
You can follow the instructions I mentioned in the above comment.